### PR TITLE
added Devschool's data

### PR DESCRIPTION
**Pull request by JimOKelly**
Please request any desired changes and I will make them. Awesome work you are doing by putting this into a repo. If you need me to remove your name in a student review I will do so as you are allowing this up in your own walled garden!

Devschool is not responsible for the students leave at course report and we don't have editorial control over them.

### CODE REVIEW

File: bootcamps/devschool/data.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+display_name: Devschool
+founded_year: 2015
+primary_email: hello@deschool.rocks
+slug: devshcool
```
Comment (rkuykendall) [inline bootcamps/devschool/data.yml]: Typo!

File: bootcamps/devschool/data.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+display_name: Devschool
+founded_year: 2015
+primary_email: hello@deschool.rocks
```
Comment (rkuykendall) [inline bootcamps/devschool/data.yml]: Typo!

**Comment by rkuykendall** [comment]
This is awesome, Jim! You're our first new bootcamp PR since we made the data public! Already noticed one typo so I'll leave notes in the PR. Also, I've gotten most of the images on the site to be retina-sharp on the /all page so you may want to use this slightly larger logo so CodeSchool isn't blurry:

codeschool_nu

File: bootcamps/devschool/description.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Your Instructor will begin by doing filling in all of your 'gaps. You will
+not just be taught to code, but to code with team-mates, and they will even
+focus on getting you to use your Mac like a pro, navigating with the
+keyboard and writing code in Vim.
+
+Towards the end of your 50 private Instructor sessions they will begin
+coaching you on how to hunt for and land paying work as a programmer, how
+to interview, and they even help you solve and understand code challenges.
+At Devschool, you graduate when you get a job, you don't graduate and then
+flounder looking for a job.
```
Comment (rkuykendall) [inline bootcamps/devschool/description.md]: For our descriptions, we're going for a third-person objective feel. Would you mind changing this, for example, from "At Devschool we combine a guided, progressive, real world approach to..." to "Devschool combines a guided, progressive, real world approach to...".

Comment (JimOKelly) [inline bootcamps/devschool/description.md]: absolutely! I will make these changes today and push up. Thanks!

---

*Jim OKelly*
Founder &
​Chief Dreamer
View my calendar
<http://t.sidekickopen04.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1p8hQMW8pTrdF5v0sYvW1qMCsg56dWP_f2XCs3T02?t=http%3A%2F%2Fteamup.com%2Fks0b1a713ec49fa70e%2F&si=5487170529001472&pi=71af327c-a63c-4b88-f934-a0d2dcd75cc1>

*Devschool​* - Learn to code, change your life!
https://devschool.rocks
(702) 799-9907
@devschoolrocks

Mailing address:
8 The Green
Suite #4595
Dover, DE 19901

<https://www.facebook.com/Devschool-1089256231104556>
<http://t.sidekickopen04.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1p8hQMW8pTrdF5v0sYvW1qMCsg56dWP_f2XCs3T02?t=https%3A%2F%2Ftwitter.com%2Fdevschoolrocks&si=5487170529001472&pi=71af327c-a63c-4b88-f934-a0d2dcd75cc1>

<http://t.sidekickopen04.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1p8hQMW8pTrdF5v0sYvW1qMCsg56dWP_f2XCs3T02?t=https%3A%2F%2Fplus.google.com%2Fb%2F117787107398081371302&si=5487170529001472&pi=71af327c-a63c-4b88-f934-a0d2dcd75cc1>

<http://t.sidekickopen04.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1p8hQMW8pTrdF5v0sYvW1qMCsg56dWP_f2XCs3T02?t=https%3A%2F%2Fwww.youtube.com%2Fchannel%2FUCBaSjb4qGVqS_Q5SVbn1hAA&si=5487170529001472&pi=71af327c-a63c-4b88-f934-a0d2dcd75cc1>

<http://t.sidekickopen04.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1p8hQMW8pTrdF5v0sYvW1qMCsg56dWP_f2XCs3T02?t=https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fdevschool&si=5487170529001472&pi=71af327c-a63c-4b88-f934-a0d2dcd75cc1>

<http://t.sidekickopen04.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJW7t5XZs1p8hQMW8pTrdF5v0sYvW1qMCsg56dWP_f2XCs3T02?t=https%3A%2F%2Fwww.pinterest.com%2Fdevschool%2F&si=5487170529001472&pi=71af327c-a63c-4b88-f934-a0d2dcd75cc1>



On Wed, Mar 23, 2016 at 7:29 AM, Robert Kuykendall <notifications@github.com

**Comment by JimOKelly** [comment]
Please let me know if these changes are good to go. This is really awesome Thinkful!

**Comment by rkuykendall** [comment]
Thanks! Going to merge in and debug, will post here when it goes live later today.

**Comment by rkuykendall** [comment]
Hey @JimOKelly, your bootcamp is live!

https://www.thinkful.com/bootcamps/devschool

https://www.thinkful.com/bootcamps/all

https://www.thinkful.com/bootcamps/city/puerto-vallarta

**Pull request merged**